src/js/camresolutionstest.js

```diff
@@ -85,7 +85,7 @@ CamResolutionsTest.prototype = {
             ' not supported');
         this.maybeContinueGetUserMedia();
```

Comment:
Can you move this.maybeContinueGetUserMedia out of the if and remove the test.done() ?

I wanted the code flow here to be simplified as in "maybeContinueGetUserMedia" is the only function to call test.done().